src/styles/index.scss

```diff
@@ -2,2 +1,10 @@
+@import './utils/variable';
+@import './blocks/stopwatch';
+
+.page {
```

Comment:
Ensure that the `.page` class is being used in your HTML. If it's not used, it should be removed to keep the stylesheet clean.